src/main/java/com/srans/nestserver/controller/InvoiceController.java

```diff
@@ -120,4 +135,73 @@ public Invoice createinvoice(@Valid @RequestBody Invoice invoice) {
 		response.put("deleted", Boolean.TRUE);
 		return response;
 	}
+
+	// Invoice generate for multiple tenant...
+	@GetMapping(value = "/invoices/{ids}")
+	@ResponseBody
+	List<Invoice> getTenantById(@PathVariable long[] ids) {
+
+		List<Invoice> invoice = new ArrayList<>();
+
+		for (long id : ids) {
+			Invoice invoiceData = new Invoice();
+
+			System.out.println(id);
```

Comment:
Remove this system and use logger to log

Reply:
Sir, I don't understand, You mean adding logger.